.travis.yml

```diff
@@ -34,8 +34,6 @@ stages:
       AND ( branch = master
             OR branch =~ ^release/[\d\.]+$
             OR tag =~ ^v[\d\.]+$ )
-- name: process snaps
-  if: repo = MirServer/mir
```

Comment:
Is this unrelated cleanup?

Reply:
Yes, the same stage is configured at the top. [`git blame`](https://github.com/MirServer/mir/blame/c64dc50f6068ae2d0d2683867f1d2311b59da8eb/.travis.yml#L37L38) doesn't explain to me how this came to be here…

Reply:
You have to look at the history.

- 09/07/18: I added it at the top
- 12/07/18: You added it at the bottom

As the duplication "worked" after merging nobody noticed before now.